tests/ui/exact-harness/fail_on_missing/subset.rs

```diff
@@ -0,0 +1,39 @@
+// Copyright Kani Contributors
+// SPDX-License-Identifier: Apache-2.0 OR MIT
+// kani-flags: --harness existing --harness check_blah --harness check_foo --exact
+//! Check that we just ignore non-matching filters
```

Comment:
This should say "Check that we error out with non-matching filters when `--exact` is used".